src/content/docs/docs/support/use-extension.mdx

```diff
@@ -0,0 +1,202 @@
+    * <LinkOut href="https://addons.mozilla.org/en-US/firefox/addon/web-monetization-extension/">Mozilla Firefox</LinkOut>
+    * <LinkOut href="https://microsoftedge.microsoft.com/addons/detail/web-monetization-extensio/imjgemgmeoioefpmfefmffbboogighjl">Microsoft Edge</LinkOut>
+    :::note[Permissions]
+    The extension will ask for permission to access your browser tabs and your data for all websites. The extension requires these permissions to check each page you visit for the Web Monetization [`<link>`](/docs/references/html-link-rel-monetization) element.
+    :::
+* You'll also need an account with a [Web Monetization-compatible wallet](/docs/resources/op-wallets) and the wallet address or payment pointer assigned to you during account set up. 
+
+## Connect the extension to your wallet and set a budget 
+
+<Steps>
+1. Open the extension and enter your wallet address or payment pointer in the given field.
+    * Example wallet address - `https://wallet.example.com/alice`
+    * Example payment pointer - `$wallet.example.com/alice`
+    <img src="/img/docs/extension/connection-setup.png" alt="Extension setup screen with wallet address and amount fields, renew monthly toggle, and connect button" style="max-width:300px" class="img-outline"/>
+2. Enter the amount to make available to the extension. 
+3. If you want the amount to renew each month, select the **Renew monthly** toggle.
+4. Click **Connect**.
+5. Click **Agree** to allow the extension to automatically connect with your wallet provider and add a security key to your wallet account.<br /><img src="/img/docs/extension/connection-wallet.png" alt="Consent screen to allow the extension to connect with the wallet provider with agree and decline buttons" style="max-width:300px" class="img-outline"/><br />If your wallet provider's website opened in your browser, proceed to the next step. If you received a connection error, review the instructions for [resolving the error](#resolve-a-wallet-connection-error).
+6. Sign in to your wallet account if you haven't already, then confirm the connection and budget amount. The look of the confirmation screen will be different for each wallet provider. Below is an example screen.<br /><img src="/img/docs/extension/example-grant.png" alt="Example budget confirmation screen with accept and decline buttons" style="max-width:400px" class="img-outline" />When the connection is successful, your browser will show a success screen.<br /><img src="/img/docs/extension/connection-success.png" alt="Web page displaying a connection successful screen" style="max-width:400px" class="img-outline"/>
+</Steps> 
+
+## Adjust your budget 
+
+<Steps>
+1. Click the cog in the upper-right of the extension.
+    <br /><img src="/img/docs/extension/extension-cog.png" alt="Extension showing the cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
```

Comment:
Can we add `width` and `height` attribute to images so they don't cause scrolling on loading?

Reply:
Or let's not do it manually, can automate it later.